pkg/apis/acid.zalan.do/v1/crds.go

```diff
@@ -167,6 +167,9 @@ var PostgresCRDResourceValidation = apiextv1.CustomResourceValidation{
 									"subPath": {
 										Type: "string",
 									},
+									"subPathExpr": {
+										Type: "string",
+									},
```

Comment:
can you reflect the oneOf part also in the Go code?

Reply:
Done.